CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -1553,7 +1553,7 @@ Features for this release were tracked via the use of the [kubernetes/features](
 * [beta] [Kubelet TLS Bootstrap](https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/#kubelet-configuration)
 
 #### Audit Logging
-* [alpha] Advanced Auditing enhances the Kubernetes API [audit logging](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-logs) capabilities through a customizable policy, pluggable audit backends, and richer audit data.
+* [alpha] Advanced Auditing enhances the Kubernetes API [audit logging](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-logs) capabilities through a customizable policy, pluggable audit backends, and richer audit data.
```

Comment:
`#audit-logs` doesn't exist anymore — we should probably put it at the base url without `#` suffix